### PR TITLE
[DH-246] bump python to 3.11

### PR DESCRIPTION
**Pull request by shaneknapp**

### CODE REVIEW

**Comment by shaneknapp** [comment]
...and now begins dependency hell.

`(ノಠ益ಠ)ノ彡┻━┻`

**Comment by shaneknapp** [comment]
there aren't enough tables in this world to express how i feel about the datahub build failure message:

```
# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "/srv/conda/lib/python3.10/site-packages/conda/exceptions.py", line 1124, in __call__
        return func(*args, **kwargs)
      File "/srv/conda/lib/python3.10/site-packages/conda_env/cli/main.py", line 78, in do_call
        exit_code = getattr(module, func_name)(args, parser)
      File "/srv/conda/lib/python3.10/site-packages/conda/notices/core.py", line 109, in wrapper
        return func(*args, **kwargs)
      File "/srv/conda/lib/python3.10/site-packages/conda_env/cli/main_update.py", line 132, in execute
        result[installer_type] = installer.install(prefix, specs, args, env)
      File "/srv/conda/lib/python3.10/site-packages/mamba/mamba_env.py", line 140, in mamba_install
        print(solver.explain_problems())
    IndexError: map::at

`$ /srv/conda/bin/mamba update -p /srv/conda -f /tmp/environment.yml`

  environment variables:
                 CIO_TEST=<not set>
  CONDA_AUTO_UPDATE_CONDA=false
                CONDA_DIR=/srv/conda
               CONDA_ROOT=/srv/conda
           CURL_CA_BUNDLE=<not set>
               LD_PRELOAD=<not set>
                     PATH=/srv/conda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin
                          :/bin:/usr/lib/rstudio-server/bin
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>

     active environment : None
       user config file : /home/jovyan/.condarc
 populated config files : /srv/conda/.condarc
          conda version : 23.1.0
    conda-build version : not installed
         python version : 3.10.10.final.0
       virtual packages : __archspec=1=x86_64
                          __glibc=2.35=0
                          __linux=6.2.0=0
                          __unix=0=0
       base environment : /srv/conda  (writable)
      conda av data dir : /srv/conda/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /srv/conda/pkgs
                          /home/jovyan/.conda/pkgs
       envs directories : /srv/conda/envs
                          /home/jovyan/.conda/envs
               platform : linux-64
             user-agent : conda/23.1.0 requests/2.28.2 CPython/3.10.10 Linux/6.2.0-1018-aws ubuntu/22.04.4 glibc/2.35
                UID:GID : 1000:1000
             netrc file : None
           offline mode : False


An unexpected error has occurred. Conda has prepared the above report.
```

**Comment by shaneknapp** [comment]
> there aren't enough tables in this world to express how i feel about the datahub build failure message:
> 
TL;DR:  apparently bumping mamba makes things work.
